databricks/koalas/tests/test_series.py

```diff
@@ -459,6 +459,31 @@ def test_fillna(self):
         self.assert_eq(kser, pser)
         self.assert_eq(kdf, pdf)
 
+    def test_dot(self):
+        pser = pd.Series([0, 1, 2, 3])
+        kser = ks.from_pandas(pser)
+
+        self.assert_eq(kser.dot(kser), pser.dot(pser))
+
+        # other_pser = pd.Series([-1, 2, -3, 4])
```

Comment:
@ueshin It seems we don't support the test case commented out for now. Are we going to support this?

Reply:
`kser.dot(other_kser)` will hit
```
ValueError: Cannot combine the series or dataframe because it comes from a different 
dataframe. In order to allow this operation, enable 'compute.ops_on_diff_frames' opti
on.
```

Reply:
Shall we use `option_context` to enable it?

```py
with ks.option_context("compute.ops_on_diff_frames", True):
    ...
```

Reply:
Got it, we don't enable `compute.ops_on_diff_frames` without user acknowledgment.